reporter/html.js

```diff
@@ -533,7 +533,8 @@ function appendTest( name, testId, moduleName ) {
 
 // HTML Reporter initialization and load
 QUnit.begin(function( details ) {
-	var qunit = id( "qunit" );
+	var qunit = id( "qunit" ),
+		tests;
```

Comment:
from the style guide, variables declared without an initializer (`= ...`) must be declared first. Also, we should use a single `var` for each variable.